### DISCUSSION TITLE
Assign issues to a specific Beta Project from a query parameter

### DISCUSSION THREAD

**Discussion by jeherve** [community/community#19811] [Projects and Issues]
GitHub currently offers a number of query parameters one can use to provide information when creating issues. This is documented on [this support page](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-url-query).

One of these parameters, `projects`, allows you to add the issue to a project board. This works well when providing the ID of a classic project. However, that does not seem to work with the new Beta projects.

Is that something that could be enabled in the future, maybe when the new projects come out of Beta and do not conflict with the classic projects anymore?

**Comment by dpasque** [comment]
Hi GitHub folks! 👋 😄  I'm wondering if you had any insights into this question, especially now that the new Projects are out of beta? Could you see this being something supported with the new Projects?

** Some background context: we do a lot of our triage and tracking with GitHub Classic Projects. We also provide a lot of users URLs to file issues, and rely on that query param to make sure the issue ends up on our triage boards. This lack of support is actually the single thing that's keeping us from switching over to using the new Projects!

**Reply by omerbensaadon** [reply]
Hey! Have you tried using the [add-to-projects](https://github.com/actions/add-to-project) action to make sure everything ends up on your board?

**Reply by dpasque** [reply]
> Have you tried using the [add-to-projects](https://github.com/actions/add-to-project) action to make sure everything ends up on your board?

Hi! 👋 Thanks for pointing out that action -- that is very helpful and we plan on using that to help get things on some of our boards! 👍  

I don't think that _fully_ mitigate our need for adding projects by URL though. I can explain a bit more in depth if that helps...

We have a lot of different repos and teams (like most folk!), and to help control the chaos of issue-reporting that can happen in those ecosystems, we maintain a "source of truth" tool that maps our products and feature areas to issue-reporting URLs. That tool is also easily modifiable so products and teams can always update their issue-reporting preferences.

We do have a few big project boards that make sense to automate with actions, but we also have a lot of smaller team-based boards (that change frequently), and I think accounting for all of those across all our repos with actions might get a little bit unwieldy! 

I hope that helps, and thanks again for the updates and the suggestions! 🙂

**Comment by dpasque** [comment]
It looks like this also might be the same question asked here: https://github.com/orgs/community/discussions/33140

**Comment by dpasque** [comment]
Hey there! 😄 Checking in again to see if the GitHub folks had any insight into this! It would be helpful to even know if this is just something that won't be supported with the new GitHub projects, as we're planning a lot around this functionality at Automattic. Thanks!

**Reply by omerbensaadon** [reply]
Howdy! sorry for the delayed response. 

We will have this at some point but its tough to say when. We need to better integrate new projects with the Repo Issue List (we would also want to show which project an issue is in from this list, like we do with Milestones), its a known deficiency but not currently one of our top priorities

**Reply by dpasque** [reply]
Hi there! 👋 No worries, thank you so much for getting back to us! 🙂 

> We will have this at some point but its tough to say when.

Thank you -- this is helpful to know! We will find some workarounds for the time being. 

And just to double check, you all don't take community engineering contributions, correct? From everything I read, it didn't seem like it, but I would be glad to help out with this change in anyway! 👍

**Reply by omerbensaadon** [reply]
Unfortunately not, when you host a big chunk of the world's open source code you've got to be a bit paranoid 😵

**Comment by j0siepy** [comment]
Checking to see if [this automated workflow](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/adding-items-automatically) solves this problem.
